tool/lib/test/unit.rb

```diff
@@ -882,6 +911,94 @@ def setup_options(opts, options)
         opts.on '--most-asserted=N', Integer, 'Show most asserted N tests' do |n|
           options[:most_asserted] = n
         end
+        opts.on '--launchable-test-reports=PATH', String, 'Report test results in Launchable JSON format' do |path|
+          require 'json'
```

Comment:
Is it necessary now? This PR seems to use only `JsonStreamWriter`.

Reply:
I use JSON module at https://github.com/ruby/ruby/pull/9757/files#diff-d58358cd56895151702a7ae7aae9bc0891f2e692a147a46f388e3abb784e354bR985.

Reply:
Ah, I see.